conda/recipe_template/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{{ '{% set name =' }} "{{ name }}" {{ '%}' }}
+
+package:
+{% raw %}  name: {{ name }} {% endraw %}
+  version: "{{ version.replace("v","") }}"
+
+source:
+  git_url: https://github.com/{{ github_repo }}.git
+  git_rev: {{ github_tag }}
+
+build:
+  number: {{ conda_build_number }}
+  run_exports:
+{% raw %}    # As robotology-superbuild packages tipically do not worry about ABI compatibility,
+    # we explicitly specify that for every patch version the ABI changes
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
+{% endraw %}
+
+requirements:
+  build:
+{# A C/C++ compiler and CMake are required by all packages #}
+{% raw %}    - {{ compiler('c') }}
+    - {{ compiler('cxx') }} {% endraw %}
+    - cmake
+    - pkg-config
+    - ninja
+{# Handle specific packages required for gl on Linux, see https://conda-forge.org/docs/maintainer/knowledge_base.html?#libgl #}
+{# libxcb, libxfixes, libxau and expat are not documented anywhere, but without those blocktest does not build correctly #}
+{% if require_opengl_linux %}{% raw %}    - {{ cdt('mesa-libgl-devel') }}
+    - {{ cdt('mesa-dri-drivers') }}
+    - {{ cdt('libselinux') }}
+    - {{ cdt('libxdamage') }}
+    - {{ cdt('libxxf86vm') }}
+    - {{ cdt('libxext') }}
+    - {{ cdt('libxcb') }}
+    - {{ cdt('libxfixes') }}
+    - {{ cdt('libxau') }}
+    - {{ cdt('expat') }}  {% endraw %}{% endif %}
+
+  host:
+{# List all dependencies just a host. Run dependenencies should be correctly set by run_exports #}
+{% for dep in dependencies %}    - {{ dep }}
+{% endfor %}
+{# Handle specific packages required for gl on Linux, see https://conda-forge.org/docs/maintainer/knowledge_base.html?#libgl #}
+{% if require_opengl_linux %}    - xorg-libxfixes {% endif %}
```

Comment:
Add the osx run stuff here I guess

Reply:
Done in https://github.com/robotology/robotology-superbuild/pull/652/commits/05593fddf73fac940540d6d48e1b181e11b7351d .